typescript/src/world-company-remuneration/basket-informations.ts

```diff
@@ -0,0 +1,46 @@
+const K = "K";
```

Comment:
This can be in its own file - reuseable by other classes

Reply:
Hello, as the repo said, this is only a support for interviewing people, we already know that they are a lot of issues in this code. Thank you for your participation anyway